docs/dns-usage.rst

```diff
@@ -0,0 +1,191 @@
+Using the API
+=============
+
+Client
+------
+
+:class:`Client <gcloud.dns.client.Client>` objects provide a means
+configure your DNS applications.  Eash instance holds both a ``project``
+and an authenticated connection to the DNS service.
+
+For an overview of authentication in ``gcloud-python``, see:
+http://gcloud-python.readthedocs.org/en/latest/gcloud-auth.html
+
+Assuming your environment is set up as described in that document,
+create an instance of :class:`Client <gcloud.dns.client.Client>`.
+
+  .. doctest::
+
+     >>> from gcloud import dns
+     >>> client = dns.Client()
+
+Override the credentials inferred from the environment by passing explicit
+``credentials`` to one of the alternative ``classmethod`` factories,
+`:meth:gcloud.dns.client.Client.from_service_account_json`:
```

Comment:
I don't think this (the above paragraph through the end of the section) is necessary since the factories are described in `gcloud-auth`.